qiita_pet/handlers/api_proxy/sample_template.py

```diff
@@ -23,6 +23,37 @@
 from qiita_pet.handlers.api_proxy.util import check_access, check_fp
 
 
+def sample_template_get_req(samp_id, user_id):
+    """Gets the json of the full sample template
+
+    Parameters
+    ----------
+    samp_id : int
+        SampleTemplate id to get info for
+    user_id : str
+        User requesting the sample template info
+
+    Returns
+    -------
+    dict of objects
+        {'status': status,
+         'message': msg,
+         'template': dict of {str: {str: object, ...}, ...}
+
+        template is dictionary where the keys access_error the metadata samples
+        and the values are a dictionary of column and value.
+        Format {sample: {column: value, ...}, ...}
+    """
+    template = SampleTemplate(int(samp_id))
```

Comment:
I think @antgonza comment here was the fact that the documentation of the function and the code are out of sync. You are describing the input as an int, but you are actually accepting ints and str that can be casted to int.